### DISCUSSION TITLE
List of an organization users for the GitHub enterprise server need to be imported

### DISCUSSION THREAD

**Discussion by KRavi0403** [community/community#123906] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Hi Team,

I need a help here to get the list of all the users in organization of GitHub enterprise server  , please do help me to get the list of users in a csv format .

Regards,
Ravi

**Comment by ebndev** [comment]
Hi @KRavi0403, 

Maybe this doc would be relevant to you: [Viewing people in your enterprise
](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/viewing-people-in-your-enterprise)

In step 6 under [this section](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/viewing-people-in-your-enterprise), you'll see the screenshot has an Export button as well.

**Comment by KRavi0403** [comment]
Hi Community/Community,

We are using the github Enterprise server and I dont have any option to
download the user based on the organization.

Regards,
Ravi

**Reply by queenofcorgis** [reply]
@KRavi0403 would you mind sharing what version of the server you're using?

**Comment by KRavi0403** [comment]
Hi Community/Community,

We are using 3.9.4 GitHub enterprise version .

Regards,
Ravi

**Reply by queenofcorgis** [reply]
@KRavi0403 the ability to export organization members is limited to Organization owners on GitHub.com and GitHub Enterprise Cloud plans.
